### PR TITLE
new: Add profile links to leaderboard.

### PR DESCRIPTION
**Pull request by twygod**

### CODE REVIEW

**Review by 0xbarbs — COMMENTED** [review]
Looks like the leaderboard page is crashing currently

File: src/components/dao/LeaderboardTableRow.jsx [hunk 1]
```diff
@@ -1,41 +1,37 @@
...
-    voteCount = 0,
-  } = data;
-
+  const { address, unionBalance = ZERO, votes = ZERO, delegatedVotes = ZERO, voteCount = 0 } = data;
+  const { chain } = useNetwork();
   return (
     <TableRow key={address}>
       <TableCell fixedSize>
-        <Avatar size={24} address={address} />
+        <Link to={`/profile/${EIP3770[chain.id]}:${data.address}`}>
```
Comment (mt-code) [inline src/components/dao/LeaderboardTableRow.jsx:15]: I think this line is causing the error when the user is not connected to a network. It is attempting to access `chain.id` for a chain that is undefined. The same thing is happening on line 20.

Suggested fix is probably to default to the `mainnet.id` if `chain` is not connected.

File: src/components/dao/LeaderboardTableRow.jsx [hunk 2]
```diff
@@ -1,41 +1,38 @@
...
-    address,
-    unionBalance = ZERO,
-    votes = ZERO,
-    delegatedVotes = ZERO,
-    voteCount = 0,
-  } = data;
-
+  const { address, unionBalance = ZERO, votes = ZERO, delegatedVotes = ZERO, voteCount = 0 } = data;
+  const { chain } = useNetwork();
+  const userLink = chain ? `/profile/${EIP3770[chain.id]}:${data.address}` : "";
```
Comment (0xbarbs) [inline src/components/dao/LeaderboardTableRow.jsx]: Instead of disabling the link if not connected can we default to Optimism?

**Comment by 0xbarbs** [comment]
Closing as leaderboard now links to tally #308

**Pull request closed without merging**